Client/Coordinators/SettingsCoordinator.swift

```diff
@@ -257,6 +258,9 @@ class SettingsCoordinator: BaseCoordinator,
     }
 
     func pressedTheme() {
+        if ReduxFlagManager.isReduxEnabled {
+            store.dispatch(ActiveScreensStateAction.showScreen(.themeSettings))
+        }
```

Comment:
I don't think this should be in the coordinator 🤔 Can this be in the VC instead?

Reply:
Ideally it should be triggered before the screen is presented or pushed but it's one of the points I want to define with Orla

Reply:
I think it's fine for the coordinator to dispatch events to the store since it's not doing any work, it's just informing the store that an navigation event has occurred.